cognite_toolkit/cdf_tk/describe.py

```diff
@@ -0,0 +1,162 @@
+            table.add_row("Last updated time", str(datetime.datetime.fromtimestamp(space.last_updated_time / 1000)))
+    try:
+        containers = client.data_modeling.containers.list(space=space_name, include_global=True, limit=None)
+    except Exception as e:
+        print(f"Failed to retrieve containers for data model {model_name}.")
+        print(e)
+        return
+    container_list = [(space_name, c.external_id) for c in containers.data]
+    containers_str = []
+    for c in container_list:
+        containers_str.append(f"{c[1]}\n")
+    if len(containers_str) > 0:
+        containers_str[-1] = containers_str[-1][0:-1]
+    table.add_row(f"Containers ({len(containers_str)})", "".join(containers_str))
+    print(table)
+    try:
+        data_model = client.data_modeling.data_models.list(
+            space=space_name, include_global=True, inline_views=True, limit=None
+        )
+    except Exception as e:
+        print(f"Failed to retrieve data model {model_name} in space {space_name}.")
+        print(e)
+        return
+    if len(data_model) == 0:
+        print(f"Failed to retrieve data model {model_name} in space {space_name}.")
+        return
+    model_name = data_model.data[0].name
+    if len(data_model.data) > 1:
+        print(f"Found {len(data_model.data)} data models in space {space_name}.")
+        print(f"  Only describing the first one ({model_name}).")
+        print("  Use the --data-model flag to specify which data model to describe.")
+
+    table = Table(title=f"Data model {model_name} in space {space_name}")
+    table.add_column("Info", justify="right")
+    table.add_column("Value", justify="left", style="green")
+    table.add_row("Description", str(data_model.data[0].description))
+    table.add_row(
+        "Version",
+        str(data_model.data[0].version),
+    )
+    table.add_row(
+        "Global",
+        "True" if data_model.data[0].is_global else "False",
+    )
+    table.add_row("Created time", str(datetime.datetime.fromtimestamp(data_model.data[0].created_time / 1000)))
+    table.add_row(
+        "Last updated time", str(datetime.datetime.fromtimestamp(data_model.data[0].last_updated_time / 1000))
+    )
+    views = data_model.data[0].views
+    table.add_row("Number of views", str(len(views)))
+    view_names = [f"{v.external_id}\n" for v in views]
+    if len(view_names) > 0:
+        view_names[-1] = view_names[-1][0:-1]
+    table.add_row("List of views", "".join(view_names))
+    print(table)
+    for view in views:
+        table = Table(title=f"View {view.external_id}, version {view.version} in space {space_name}")
+        table.add_column("Info", justify="left")
+        table.add_column("Value", justify="left", style="green")
+        table.add_row("Number of properties", str(len(view.properties)))
+        table.add_row("Used for", str(view.used_for))
+
+        implements_str = [f"{i}\n" for i in view.implements or []]
```

Comment:
Should we do the same as for views and containers?
```suggestion
        implements_str = [f"{i.external_id}\n" for i in view.implements or []]
```